Sources/RInAppMessaging/Repositories/ConfigurationRepository.swift

```diff
@@ -46,6 +46,6 @@ internal class ConfigurationRepository: ConfigurationRepositoryType {
     }
 
     func getConfigEndpointURL() -> String? {
```

Comment:
Maybe we should rename `getConfigEndpointURL` to `getConfigEndpointURLString`.
Otherwise `getConfigEndpointURL` can return `URL?` instead of `String?` if possible.

Reply:
ok. Yeah I think it makes sense